packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -194,10 +198,6 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
       };
 
       before(async function () {
-        // Restart the relay to reset the limits
```

Comment:
why is this deleted?

Reply:
There’s no longer a need to restart the Local Relay, as it’s now a standalone CI. When the CI starts, the Local Relay is already a fresh instance.

Also, `await cacheService.clear(requestDetails);` was removed because the original cache is needed for the preconfigured plans test. Additionally, IP addresses are no longer automatically linked to BASIC plans, so clearing the cache is no longer necessary.

Reply:
perfect :)